packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -308,7 +310,7 @@ async function updateWorkflowName() {
 
 // eslint-disable-next-line
 const _saveWorkflow = async () => {
-	await workflowService.updateWorkflow(wf.value.dump());
+	await workflowService.updateWorkflow(wf.value.dump(), currentProjectId.value ?? undefined);
```

Comment:
Can we just use `activeProjectId` directly instead of through an additional var "currentProjectId"?

Reply:
no, because once we hit the `onUnmounted` the active project has been cleared away (This is why the injection in `api.ts` is failing)